br/pkg/lightning/tikv/tikv.go

```diff
@@ -253,13 +230,16 @@ func CheckPDVersion(ctx context.Context, tls *common.TLS, pdAddr string,
 }
 
 // CheckTiKVVersion checks the version of TiKV.
-func CheckTiKVVersion(ctx context.Context, tls *common.TLS, pdAddr string,
-	requiredMinVersion, requiredMaxVersion semver.Version) error {
+func CheckTiKVVersion(
+	ctx context.Context,
+	pdHTTPCli pdhttp.Client,
+	requiredMinVersion, requiredMaxVersion semver.Version,
+) error {
 	return ForAllStores(
 		ctx,
-		tls.WithHost(pdAddr),
-		StoreStateDown,
-		func(c context.Context, store *Store) error {
+		pdHTTPCli,
+		metapb.StoreState_Offline,
```

Comment:
offline == down?

Reply:
In the old code's comment, the order is `Tombstone < Offline < Down < Disconnected < Up`. Now we only have `Tombstone >  Offline > Up`, I should have made correct order translation.